hdf5.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
```

Comment:
This file already exists in config/cmake/libhdf5.pc.in. Also a fortran one. Both could be moved up into the config folder.